pkg/trace/api/api.go

```diff
@@ -497,6 +508,23 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 		}()
 	}
 }
+func extractDroppedP0sHeaders(req *http.Request, ts *info.TagStats) int64 {
```

Comment:
You need an empty line above. 
```suggestion
// droppedP0Traces returns the number of traces dropped by the client, as reported in the HTTP header h.
func droppedP0Traces(h http.Header, ts *info.TagStats) int64 {
```
Wouldn't this be better? It describes the return value. The current name makes it sounds like we are extracting headers. Instead, we are extracting a count of traces _from the headers_.